docs/readmes/howtos/inbound_roaming.md

```diff
@@ -7,7 +7,6 @@ hide_title: true
 *Last Updated: 4/08/2021*
 
```

Comment:
I think these are good updates, but can we make the following statement (perhaps in a table of sorts?):

* Local Non-Federated + Roaming = 1 feg_lte network (AGW), 1 feg network (Roaming FeG)
* Local Federated + Roaming = 1 feg_lte network (AGW) and 2 feg networks (Local FeG + Roaming FeG)

Reply:
what about this

![image](https://user-images.githubusercontent.com/16157139/122658448-4b753d80-d122-11eb-81b6-caa23c2be0bb.png)

Reply:
Can we transpose this? 
Also, recommending to remove "Network Type" and `swagger name` and using the NMS nomenclature instead. 

`lte_feg` --> `feg_lte`
`fed` --> `feg`
```
_____________________ ____________________ ______________. _______________
|Network Type    --->      | `feg` roaming              | `feg` local          | `feg_lte` (AGW)|
------------------------  ----------------------  ----------------- -----------------
| Local - Non Federated  |                  Yes               |            No            |         Yes              |
| Local - Federated          |                  Yes               |            Yes           |        Yes               |

```

Reply:
sure! not sure where i checked the end point names, but htey were all wrong 😭

![image](https://user-images.githubusercontent.com/16157139/122810182-e6395d80-d283-11eb-8b0a-efd985600b45.png)

Reply:
After reviewing the table, i think the original is more complete. 

My idea with that table is to show the gateway needed in each network. For example: any configuration requieres a feg_lte network that will be connected to the local feg network. 

I think in the purposed table (see below), we are mixing concepts of networks and gateways: For example on `Non-Federated` we still need a local feg network. But that local feg network do not need a local feg gateway defined. The local feg network is a `virtual` network that will be doing the routing to the roaming network 




![image](https://user-images.githubusercontent.com/16157139/122812462-8c866280-d286-11eb-99f1-4ef1cd04d74f.png)

Reply:
ok i think i got the idea after checking side to side both tables.
Let me do some changes to convince myself :D

Reply:
Changes done, note I have moved the table to the `Create a Federated Deployment` section. 
I tooks like that descriptio belongs there better

Reply:
lgtm!